dces-drc-integration/build.gradle

```diff
@@ -59,6 +59,8 @@ dependencies {
 	implementation platform("io.sentry:sentry-bom:$versions.sentryVersion")
 
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'org.apache.tomcat.embed:tomcat-embed-core:10.1.34'
```

Comment:
This whole line should be unnecessary now.

Reply:
Yes, not sure how that was still there. Now removed.